src/domain/eventSearch/filterSummary/FilterSummary.tsx

```diff
@@ -70,28 +78,19 @@ const FilterSummary: React.FC<Props> = ({ onClear, route }) => {
 
   const handleFilterRemove = (value: string, type: FilterType) => {
     const search = getSearchQuery({
-      categories:
-        type === 'category'
-          ? categories.filter((category) => category !== value)
-          : categories,
-      dateTypes:
-        type === 'dateType'
-          ? dateTypes.filter((dateType) => dateType !== value)
-          : dateTypes,
-      divisions:
-        type === 'division'
-          ? divisions.filter((division) => division !== value)
-          : divisions,
+      categories: getFilteredList(type, 'category', value, categories),
+      hobbyTypes: getFilteredList(type, 'hobbyType', value, hobbyTypes),
+      dateTypes: getFilteredList(type, 'dateType', value, dateTypes),
+      divisions: getFilteredList(type, 'division', value, divisions),
```

Comment:
I think keeping getFilteredList would have been better inside the handleFilterRemove because tou don't have to pass four parameters, makes it little bit harder to understand

Reply:
fixed